pt_BR/readme.md

```diff
@@ -0,0 +1,750 @@
+   return new Promise(/* ... */);
+});
+```
+
+Os hooks `beforeEach` & `afterEach` podem compartilhar o contexto com o teste:
+
+```js
+test.beforeEach(t => {
+	t.context.data = generateUniqueData();
+});
+
+test(t => {
+	t.is(t.context.data + 'bar', 'foobar');
+});
+```
+
+O contexto é, por padrão, um objeto, mas também pode ser atribuído diretamente:
+
+```js
+test.beforeEach(t => {
+	t.context = 'unicorn';
+});
+
+test(t => {
+	t.is(t.context, 'unicorn');
+});
+```
+
+### Encadeamento de modificadores de teste
+
+Você pode encadear modificadores de teste das seguintes maneiras:
+
+```js
+test.before.skip([title], testFn);
+test.skip.after(....);
+test.serial.only(...);
+test.only.serial(...);
+```
+
+Isso é especialmente útil temporariamente usando `skip` ou `only` em um teste, sem perder a informação e o comportamento que os outros modificadores fornecem.
+
+### Módulo personalizado de asserção
+
+Você pode usar qualquer módulo de asserção no lugar, ou além, do que vem com o AVA, mas você não será capaz de usar o método `.plan()`, [ainda](https://github.com/sindresorhus/ava/issues/25).
+
+```js
+import assert from 'assert';
+
+test(t => {
+	assert(true);
+});
+```
+
+### Suporte para ES2015
+
+AVA vem com suporte embutido para ES2015 através do [Babel 6](https://babeljs.io). Basta escrever seus testes em ES2015. Nenhuma configuração extra é necessária. Você pode usar qualquer versão do Babel em seu projeto. Usamos nosso próprio Babel empacotado com o [`es2015`](http://babeljs.io/docs/plugins/preset-es2015/) e predefinições [`stage-2`](http://babeljs.io/docs/plugins/preset-stage-2/).
+
+#### Transpiling de Módulos Importados
+
+AVA atualmente só faz o transpiling de testes que você pede para serem executados. *Não fará o transpiling de módulos que você importa (```import```) de fora do teste.*  Embora existam razões válidas para esta abordagem, pode não ser o que você espera!
+
+Como uma solução simples, você pode usar [Babel's require hook](https://babeljs.io/docs/usage/require/) a fim de fazer transpiling on-the-fly de módulos que são posteriormente importados. Como o AVA oferece suporte para a sintaxe de módulo ES2015, você pode usá-lo para importar o hook requerido em si:
+
+```js
+import test from 'ava';
+import 'babel-core/register';
+import foo from './foo'; // <-- foo can be written in ES2015!
+
+test('foo bar', t => {
+	t.same('baz', foo('bar'));
+});
+```
+
+[#111](https://github.com/sindresorhus/ava/issues/111) está rastreando este item como um melhoria em potencial.
+
+### Suporte para Promessas
+
+Se você retornar uma promessa no teste, você não precisa explicitamente encerrá-lo, já que se encerrará quando a promessa for resolvida.
+
+```js
+test(t => {
+	return somePromise().then(result => {
+		t.is(result, 'unicorn');
+	});
+});
+```
+
+### Suporte para função geradora
+
+AVA vem com suporte embutido para [funções geradoras](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*).
```

Comment:
Use link Brazilian Portuguese is better (https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Statements/function*)